src/explore-education-statistics-admin/src/services/applicationInsightsService.ts

```diff
@@ -0,0 +1,25 @@
+import { ApplicationInsights } from '@microsoft/applicationinsights-web';
+import client from '@admin/services/util/service';
+
+const service = {
+  GetInsightsKey(): Promise<string> {
```

Comment:
lowercase the first "g" here - it's the javascript convention to write function names as camel-cased.